subsystem/wrist.py

```diff
@@ -0,0 +1,64 @@
+import config
+import constants
+from toolkit.subsystem import Subsystem
+from toolkit.motors.rev_motors import SparkMax, SparkMaxConfig
+from units.SI import radians
+import rev
+from math import pi
+import math
+
+class Wrist(Subsystem):
+    def __init__(self):
+        super().__init__()
+        self.wrist_motor = SparkMax(
+            can_id=config.wrist_motor_id, inverted=True,
+            config=config.WRIST_CONFIG
+        )
+
+    def init(self):
+        self.wrist_motor.init()
+        self.wrist_motor.motor.setClosedLoopRampRate(constants.wrist_time_to_max_vel)
+        self.wrist_abs_encoder = self.wrist_motor.motor.getAbsoluteEncoder(
+            rev.SparkMaxAbsoluteEncoder.Type.kDutyCycle
+        )
+    
+    def set_wrist_angle(self, pos: float):
```

Comment:
One really quick thing is to change `pos: float` to `angle: radians` just to not get confused

you already imported the radians value so you can just switch that